camel/CSVExample/src/main/java/ibmi/example/CSVExample.java

```diff
@@ -0,0 +1,178 @@
+
+        /**
+         * These values below exist if you are not journaling your IBM i table that you are querying
+         */
+        //localDS.setTransactionIsolation("none");
+        
+        /**
+         * This binds the localDS we set above to the route jt400
+         */
+        context.getRegistry().bind("jt400", localDS);
+
+        /**
+         * Setting our Marshall options for CSV
+         */
+		
+
+        /**
+         * What example do we want to accomplish? Set the variable above to choose the example 
+         */
+        context.addRoutes(new RouteBuilder() {
+            @Override
+            public void configure() {
+                from("timer://ibmiexamplecheckinterval?period=5000")
+                        /**
+                         * This is our example query. Here we are pulling results from the system.
+                         * You set the body of our program (which is a mutable object used to contain the results throughout).
+                         * Here I have set it to a constant, but you can use the simple() function to set some dynamic values
+                         * if needed in your own example. Once you have set the body, you send to JT400. The results are then 
+                         * stored in our body, replacing the query we set it to before.  
+                         */
+                          .setBody(constant("select * from QSYS2.NETSTAT_INFO"))
+                          .to("jdbc:jt400")
+                        /**
+                         * Now we have queried some results! We want to send it to a CSV for output. 
+                         */                
+                        .to("direct:" + whatexample);
+            }
+        });
+
+        /**
+         * This will do a standard output of values to CSV
+         */
+        context.addRoutes(new RouteBuilder() {
+            @Override
+            public void configure() {
+                // The from should match our to in the calling route. This connects our two routes
+                from("direct:standardcsvexample")
+                // Marshall is how we are going to divide up each result from the row and what to do
+                .marshal()
+                // This creates our CSV from the results that were marshalled
+                .csv()
+                // This specific header we are setting is the name of our file
+                .setHeader("CamelFileName", constant("CSVExample.csv"))
+                // This is where we are storing the file. This is set at the top by constants
+            	.to("file:" + filelocation);
+            }
+        });
+      
+        /**
+         * This will show how to adjust the pipe delimiter
+         */
+        context.addRoutes(new RouteBuilder() {
+            @Override
+            public void configure() {
+                from("direct:pipedelimitedexample")
```

Comment:
This is the one I need help with making. When I add Pipes the double quotes are still there for strings. 

Where this gets tricky is we have a lot of examples that have single and double quotes within the examples, which is why we use the pipe. So how to have a pipe delimited example with no double quotes around strings

Example of what we want

TEST|THIS IS A STRING AS WELL|ANOTHER STRING

What we get currently
"TEST"|"THIS IS A STRING AS WELL"|"ANOTHER STRING"

This also means you cannot trim a string, because the quotes are in there

Reply:
I hope to find time this week to implement this